packages/integrations/vercel/test/edge-middleware.test.js

```diff
@@ -19,17 +19,11 @@ describe('Vercel edge middleware', () => {
 	});
 
 	// TODO: The path here seems to be inconsistent?
-	it.skip('with edge handle file, should successfully build the middleware', async () => {
+	it.skip('without edge handle file, should successfully build the middleware', async () => {
```

Comment:
Still using `it.skip`, should it?

Reply:
Yes. For some reasons that I don't understand, the path of the emitted files inside the fixtures is not predictable.